lib/bora/stack.rb

```diff
@@ -316,7 +316,12 @@ def run_cfndsl(template_file, params, pretty_json)
     end
 
     def cfn_options_from_stack_config
-      valid_options = ["capabilities"]
+      valid_options = %w(capabilities tags)
+
+      # Expand Tags to key, value pairs
+      if @stack_config[:tags]
+        @stack_config[:tags] = @stack_config[:tags].collect { |k, v| { key: k, value: v } }
+      end
```

Comment:
Mutating `@stack_config` doesn't feel quite right here. Creates up side effects, for example what if this method happened to be called twice? Maybe assign the result of the "select" on the next line below to a temporary var, and expand the tags on that?